feature-authentication/src/main/java/com/apk/herbiary/screens/login/ui/SignUpButton.kt

```diff
@@ -0,0 +1,31 @@
+package com.apk.herbiary.screens.login.ui
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material.Text
+import androidx.compose.material.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.apk.herbiary.feature.authentication.R
+
+@Composable
+fun SignUpButton(modifier: Modifier = Modifier) {
```

Comment:
We can make one master button with the clickable action as parameter so it can be used in all the application